libbeat/processors/add_kubernetes_metadata/kubernetes_test.go

```diff
@@ -86,3 +87,51 @@ func TestAnnotatorDeepUpdate(t *testing.T) {
 		},
 	}, event.Fields)
 }
+
+// Test metadata are not included in the event
+func TestAnnotatorWithNoKubernetesAvailable(t *testing.T) {
+	cfg := common.MustNewConfigFrom(map[string]interface{}{
+		"lookup_fields": []string{"kubernetes.pod.name"},
+	})
+	matcher, err := NewFieldMatcher(*cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	processor := kubernetesAnnotator{
+		cache: newCache(10 * time.Second),
+		matchers: &Matchers{
+			matchers: []Matcher{matcher},
+		},
+		kubernetesAvailable: false,
+	}
+
+	processor.cache.set("foo", common.MapStr{
+		"pod": common.MapStr{
+			"labels": common.MapStr{
+				"dont":     "replace",
+				"original": "fields",
+			},
+		},
+	})
```

Comment:
Why initializing the cache if kubernetes is not available?

Reply:
Maybe it is not required. It is just a left-over from copying the above test. :)

Reply:
Ok, try to remove it then :slightly_smiling_face:

Reply:
👍